docs/source/transaction_family_specifications/supplychain_transaction_family.rst

```diff
@@ -0,0 +1,255 @@
+
+For "OwnRecords" the value 1 means Held and Owned, 0 means Owned and not held.
+
+For "OpenApplications" indicates the agent has an application open against a record and is waiting for a response.
+
+For "AcceptedApplications" indicates the agent has an application accepted against a record and can now create a child. The value is the sensor ID associated with the record.
+
+
+Record
+------
+A record represents a good or material that is being tracked in the system.
+
+.. code-block:: javascript
+
+    {
+        RecordInfo: {
+            Owner: agent_address,  // the owner of the item
+            Custodians: [],  // A list of custodians(identified by address) for the record.
+            CurrentHolder: agent_address,  // The agent in possession of the item, either the Owner or the current Custodian
+            Timestamp: timestamp,  // time stamp of record creation, seconds from the epoc
+            Sensor: 'sensor_id',  // identifier of sensor attached to item
+            Final: true | false,  // once set true the record cannot be updated
+            ApplicationFrom: agent_address,  //  The address of the applicant
+            ApplicationType: "Owner" | "Custodian",  // Which field is the application for.
+            ApplicationTerms: "text",  // human readable terms of the transfer application.
+            ApplicationStatus:  "Open" | "Rejected" | "Cancelled"
+        }
+    }
+
+
+Addressing
+----------
+
+When a setting is read or changed, it is accessed by addressing it using the following algorithm:
+
+The SCTP uses the Supplychain namespace computed as the first 6 hex characters of:
+.. code-block:: javascript
+
+    namespace = sha512('Supplychain')).hexdigest()[:6]
+
+Agent addresses are generated sha512(agent_pub_key), prepended with the Supplychain namespace.
+
+.. code-block:: javascript
+
+    agent_addr = namespace + sha512(agent_pub_key)).hexdigest()
+
+Record addresses are generated sha512 of the initial record data, prepended with the Supplychain namespace.
+
+.. code-block:: javascript
+
+    record_addr = namespace.sha512(initial_record_data)).hexdigest()
+
+
+Transactions
+============
+
+Transaction Headers
+-------------------
+
+The settings for the common Sawtooth Transaction header fields.
+
+.. code-block:: javascript
+
+    {
+        family_name: "sawtooth_supplychain"
+        family_version: "0.5"
+        encoding: "application/json"
+    }
+
+Inputs and Outputs
+++++++++++++++++++
+The inputs for config family transactions must include:
+the address of the object the transaction is operating on
+if it is a record operation it must contain any agents involved in the operation.
+The outputs for config family transactions must include:
+the address of the object the transaction is operating on
+if it is a record operation it must contain any agents involved in the operation.
+
+Dependencies
+++++++++++++
+    None
```

Comment:
presumably you would need dependencies for dependent transactions generated in sequence (create record, create application)